schemas/exchange/min-meta/json-min-db-pdbx_core-col-pdbx_core_polymer_entity.json

```diff
@@ -3251,6 +3251,28 @@
                   }
                ]
             },
+            "uniprot_ids": {
+               "type": "array",
+               "items": {
+                  "type": "string",
+                  "examples": [
+                     "P01116",
+                     "P69905"
+                  ],
+                  "description": "Uniprot accession codes assigned to polymeric entities.",
+                  "rcsb_description": [
+                     {
+                        "text": "Uniprot accession codes assigned to polymeric entities.",
+                        "context": "dictionary"
+                     },
+                     {
+                        "text": "UniProtKB Accession Number(s)",
+                        "context": "brief"
```

Comment:
We don't need to define a "brief" description, this attribute isn’t used in the UI

Reply:
@valasatava This has been fixed.